src/components/journeys/JourneyInstancesDataTable/getStaticColumns.tsx

```diff
@@ -0,0 +1,77 @@
+import dayjs from 'dayjs';
+import { GridColDef } from '@mui/x-data-grid-pro';
+
+import { ColumnNames } from './getColumns';
+import ZetkinPerson from 'components/ZetkinPerson';
+import { ZetkinJourney, ZetkinPerson as ZetkinPersonType } from 'types/zetkin';
+
+// Name concatenation
+const getPeopleString = (people: ZetkinPersonType[]) =>
+  people.map((person) => `${person.first_name} ${person.last_name}`).join(', ');
+
+export const getStaticColumns = (
+  journey: ZetkinJourney,
+  columnNames: ColumnNames
+): GridColDef[] => {
+  const staticColumns: GridColDef[] = [
+    {
+      field: 'id',
+      valueFormatter: (params) => {
+        return `${journey.singular_name} #${params.value}`;
+      },
+    },
+    {
+      field: 'people',
+      valueGetter: (params) =>
+        getPeopleString(params.value as ZetkinPersonType[]),
+    },
+    {
+      field: 'created_at',
+      type: 'date',
+      valueFormatter: (params) =>
+        dayjs(params.value as string).format('MMMM D, YYYY'),
+    },
+    {
+      field: 'updated_at',
+      type: 'date',
+      valueFormatter: (params) => dayjs(params.value as string).fromNow(),
```

Comment:
We already have `<FormattedRelativeTime>` (and the underlying `intl.formatRelative…` function) that does this. Do you reckon we could use that instead here to be consistent?

Reply:
THanks for reminding 👍